apps/passport-server/test/generic-issuance/pipelines/auto-issuance/autoIssuanceSpec.ts

```diff
@@ -0,0 +1,1089 @@
+        const attendeeTicketsAfterEnd = await requestTicketsFromPipeline(
+          pipeline.issuanceCapability.options.feedFolder,
+          autoIssuanceTicketFeedUrl,
+          pipeline.issuanceCapability.options.feedId,
+          ZUPASS_EDDSA_PRIVATE_KEY,
+          pretixBackend.get().autoIssuanceOrganizer.autoIssuanceAttendeeEmail,
+          AutoIssuanceAttendeeIdentity
+        );
+        expectLength(
+          attendeeTicketsAfterEnd.map((t) => t.claim.ticket),
+          (1 + 4) * 2 // 1 ticket + pod + 4 vouchers + 4 voucher pods
+        );
+
+        MockDate.set(savedDate);
+      }
+
+      const attendeeTicket = attendeeTickets[0];
+      const attendeeFoodVoucherTicket = attendeeTickets[1];
+      expectIsEdDSATicketPCD(attendeeFoodVoucherTicket);
+      expectToExist(attendeeTicket);
+      expectIsEdDSATicketPCD(attendeeTicket);
+      expect(attendeeTicket.claim.ticket.attendeeEmail).to.eq(
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceAttendeeEmail
+      );
+      expect(attendeeTicket.claim.ticket.attendeeName).to.eq(
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceAttendeeName
+      );
+
+      const attendeePODTicket = attendeeTickets[5];
+      expectToExist(attendeePODTicket);
+      expectIsPODTicketPCD(attendeePODTicket);
+      expect(attendeePODTicket.claim.ticket.attendeeEmail).to.eq(
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceAttendeeEmail
+      );
+      expect(attendeePODTicket.claim.ticket.attendeeName).to.eq(
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceAttendeeName
+      );
+
+      const bouncerTickets = await requestTicketsFromPipeline(
+        pipeline.issuanceCapability.options.feedFolder,
+        autoIssuanceTicketFeedUrl,
+        pipeline.issuanceCapability.options.feedId,
+        ZUPASS_EDDSA_PRIVATE_KEY,
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceBouncerEmail,
+        AutoIssuanceBouncerIdentity
+      );
+      expectLength(bouncerTickets, 2);
+      const bouncerTicket = bouncerTickets[0];
+      expectToExist(bouncerTicket);
+      expectIsEdDSATicketPCD(bouncerTicket);
+      expect(bouncerTicket.claim.ticket.attendeeEmail).to.eq(
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceBouncerEmail
+      );
+      expect(bouncerTicket.claim.ticket.attendeeName).to.eq(
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceBouncerName
+      );
+      const bouncerPODTicket = bouncerTickets[1];
+      expectToExist(bouncerPODTicket);
+      expectIsPODTicketPCD(bouncerPODTicket);
+      expect(bouncerPODTicket.claim.ticket.attendeeEmail).to.eq(
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceBouncerEmail
+      );
+      expect(bouncerPODTicket.claim.ticket.attendeeName).to.eq(
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceBouncerName
+      );
+
+      const autoIssuanceCheckinRoute =
+        pipeline.checkinCapability.getCheckinUrl();
+
+      {
```

Comment:
food vendor ticket holder test:
- can check in food voucher
- can't check in GA ticket